tools/export/uvision/uvision.tmpl

```diff
@@ -369,8 +369,8 @@
             <uSurpInc>0</uSurpInc>
             <uC99>1</uC99>
             <useXO>0</useXO>
-            <v6Lang>4</v6Lang>
-            <v6LangP>2</v6LangP>
+            <v6Lang>6</v6Lang>
+            <v6LangP>4</v6LangP>
```

Comment:
The uvision template change has to be conditional because the online compiler needs to be able to export older projects that cannot use the newer language standards. The online compiler will use the latest copy of the tools but switch the profiles accordingly so the correct language version is used.

Reply:
Reworked to parse the `-std` option. And I realised that we can actually work around uVision not knowing about `gnu++14` - it's armclang's default and we can ask for `<default>`.

Reply:
Except that workaround doesn't seem to work. Contrary to uVision documentation, if you put `<default>` in, it explicitly selects `-std=gnu11` for C and `-std=gnu++89` for C++.

(Those were the defaults for Compiler 6.9, I guess, but the documentation clearly says that it will use the compiler default by not passing an option).